index.html

```diff
@@ -163,88 +117,31 @@
-        should consider how this information might be correlated with other
-        information and the privacy risks that might be created. The potential
-        risks of collection of such data over a longer period of time should
-        also be considered.
-      </p>
-      <p>
-        Variations in implementation light level values as well as event firing
-        rates offer the possibility of fingerprinting to identify users.
-        Browser implementations may reduce the risk by limiting event rates
-        available to web application developers.
-      </p>
-      <p>
-        If the same JavaScript code using the API can be used simultaneously in
-        different window contexts on the same device it may be possible for
-        that code to correlate the user across those two contexts, creating
-        unanticipated tracking mechanisms.
-      </p>
-      <p>
-        Browser implementations should consider providing the user an
-        indication of when the sensor is used and allowing the user to disable
-        sensing.
-      </p>
-      <p>
-        Web application developers that use this specification should perform a
-        privacy assessment of their application taking all aspects of their
-        application into consideration.
-      </p>
-      <p>
-        The event defined in this specification is only fired in the
-        <a>top-level browsing context</a> to avoid the privacy risk of sharing
-        the information defined in this specification with contexts unfamiliar
-        to the user. For example, a mobile device will only fire the event on
-        the active tab, and not on the background tabs or within iframes.
+        There are no specific security and privacy considerations
+        beyond those describe in the Generic Sensor API [[GENERIC-SENSOR]].
       </p>
     </section>
     <section>
       <h2>
-        Device Light
+        Model
       </h2>
       <section>
         <h2>
-          Extensions to <a>Window</a> Interface
+          <a>AmbientLightSensor</a> Interface
         </h2>
-        <p>
-          The HTML5 specification [[!HTML5]] defines a <a>Window</a> interface,
-          which this specification extends:
-        </p>
         <pre class="idl">
-          partial interface Window {
-                          attribute EventHandler ondevicelight;
-          };
-        </pre>
-        <p>
-          The <code>ondevicelight</code> event handler and its corresponding
-          event handler event type <a>devicelight</a> MUST be supported as an
-          IDL attribute by all objects implementing the <a>Window</a>
-          interface.
-        </p>
-      </section>
-      <section>
-        <h2>
-          <a>DeviceLightEvent</a> Interface
-        </h2>
-        <pre class="idl">
-          dictionary DeviceLightEventInit : EventInit {
-              unrestricted double value;
+          [Constructor(optional SensorOptions sensorOptions)]
```

Comment:
Given that we already have a inheritance relationship (rather than a mixin/traits one) for the `Sensor` object, it seems logical to have the same for the `Sensor` options object, no?